charters/bcdc-cloud-computing-environment.md

```diff
@@ -0,0 +1,42 @@
+# BCDC Cloud Computing Environment (Terra)
+## Description
+We connect, develop, add value to, and train our community around the BCDC Cloud Computing Environment. This is a cloud-native environment that enables many functionality including cloud storage, scalable data processing pipelines, and horizontally scalable data-science environments.
+
+# Roles
+[Principal Investigator] Timothy Tickle (ttickle@broadinstitute.org)   
+[Pipelines Development Product Manager] Kylee Degatano (email@email.com)   
+[Customer Success and Education] Salin Thomas (salin@broadinstitute.org)  
+[Project Manager] Cara Mason (cara@broadinstiute.org)
+
+## Definitions
+**pipeline:** A collection of one or more functional tasks that operate on input data and, from that, transform the input data, or derive features often used to interpret the input data. In a high throughput setting, these tasks are often automated to be performed in a batch setting.
+
+## Objectives
+- To create and maintain a cloud native, horizontally scalable environment for use by the BICCN Community.   
```

Comment:
Is this limited to processing of molecular *omics data, or could they cover other modalities in the future? (I think the current members know this, but a new member might not)

Reply:
will state and clarify potential other areas.